File: 기초-브론즈 문제/백준_5543_상근날드/상근날드 .cs
using System;
using static System.Console;

namespace cs_algo
{
    class Program
    {
        public static void  Main(string[] args)
        {
            int min1 = 2000;
            for (int i = 0; i < 3; i++)
            {
                int burger = int.Parse(ReadLine());
                if (burger < min1) min1 = burger;
            }

            int min2 = 2000;
            for (int i = 0; i < 2; i++)
            {
                int drink = int.Parse(ReadLine());
                if (drink < min2) min2 = drink;
            }

            WriteLine(min1 + min2 - 50);


        }
    }
}

File: 기초 사고력문제/백준_5543_상근날드/상근날드 .py
min1 = 2000
for i in range(3):
    burger = int(input())
    if burger < min1: min1 = burger
min2 = 2000
for i in range(2):
    drink = int(input())
    if drink < min2: min2 = drink

print(f'{min1 + min2 -50}')